studio/components/interfaces/Settings/Addons/Addons.tsx

```diff
@@ -21,13 +21,17 @@ import { Alert, Button, IconChevronRight, IconExternalLink } from 'ui'
 import { ComputeInstanceSidePanel, CustomDomainSidePanel, PITRSidePanel } from './'
 import { GenericSkeletonLoader } from 'components/ui/ShimmeringLoader'
 import AlertError from 'components/ui/AlertError'
+import { useProjectContext } from 'components/layouts/ProjectLayout/ProjectContext'
 
 const Addons = () => {
   const { isDarkMode } = useTheme()
   const { ref: projectRef } = useParams()
   const snap = useSubscriptionPageStateSnapshot()
   const projectUpdateDisabled = useFlag('disableProjectCreationAndUpdate')
 
+  const { project: selectedProject } = useProjectContext()
+  const cpuArchitecture = selectedProject?.cloud_provider === 'FLY' ? 'x86 64-bit' : 'ARM'
```

Comment:
Nit: Could extract that to a utils function, so we have that stored centrally instead of repeating the logic three times

Since it's a simple ternary, just a nit really

Reply:
Extracted it into a function - might come in useful further down the line